src/data/Cameras.jsx

```diff
@@ -0,0 +1,144 @@
+var Cameras = {
+  "all" : [
+    {"ID":"C01","latitude":"-18.9325","longitude":"34.3621","veg_type":"Mixed Savanna and Woodland","dist_water_m":195,"dist_humans_m":1195},
```

Comment:
A suggestion - this data could be kicked out to a `.json` file that you `require` in to this.

Another one - this file should probably be a `.js` file, as it's not actually using `.jsx`